app/components/common/Header/NavBar/index.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+import theme from '@/styles/theme';
+import { usePathname, useRouter } from 'next/navigation';
+import styled from 'styled-components';
+
+interface NavDataProps {
+  title?: string;
+  path?: string;
+}
+interface NavProps {
+  type?: 'top' | 'main';
+  data?: NavDataProps[];
+}
+const MultiNavBar = ({ type = 'top', data }: NavProps) => {
+  const router = useRouter();
+  const path = usePathname();
+  return (
+    <NavWrapper type={type}>
+      {data &&
+        data.map((navData, idx) => (
+          <ButtonWrapper
+            onClick={() => router.push('/' + navData.path)}
+            key={idx}
+            type={type}
+            active={path === '/' + navData.path ? 'true' : 'false'}
+          >
+            {navData.title}
+          </ButtonWrapper>
+        ))}
+      {type === 'top' ? ( // 알람과 modal창을 띄워야해서 따로 분리 했습니다.
+        <>
+          <ButtonWrapper
+            type={type}
+            onClick={() => console.log('알람 이벤트 처리')}
```

Comment:
```suggestion
            onClick={() => null}
```
console.log은 확인용으로만 사용하시고, push할때는 없애주세요!
불필요한 코드를 지워야 훨씬 깔끔합니다.

Reply:
앗 넵! 주석으로 달거나 null처리하겠습니다